notebooks/pixiedust_facebook_analysis.ipynb

```diff
@@ -199,7 +199,9 @@
     "import pixiedust\n",
     "\n",
     "# Suppress some pandas warnings\n",
-    "pd.options.mode.chained_assignment = None  # default='warn'"
+    "pd.options.mode.chained_assignment = None  # default='warn'",
+    "# Suppress request warnings for not verify SSL certificates\n",
```

Comment:
tense seems off a little.  s/verify/verifying/  or could be something simpler like "Suppress SSL warnings"